examples/conflicts/server.js

```diff
@@ -5,6 +5,7 @@ const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-serv
 
 const { conflictHandler } = require('../../packages/apollo-voyager-conflicts')
 
+conflictHandler.enableLogging(console)
```

Comment:
I'm just wondering. conflictHandler (VersionedObjectState) is actually a class. It seems we never create an instance and are somehow statically accessing methods like `hasConflict`, `nextState` etc.

I think maybe we should rename the exported class to `ConflictHandler` and then we should be creating an instance like this:

```js
const conflictHandler = new ConflictHandler({ logger: console })
```

WDYT?

Reply:
This is singleton created just for users  convenience. Sadly we cannot create interface constructor method. This way we are exposing default option without asking people to create object.